packages/app-mobile/contentScripts/markdownEditorBundle/contentScript.ts

```diff
@@ -47,6 +48,10 @@ export const createEditorWithParent = ({
 		throw new Error(`Unable to find parent element for editor (class name: ${JSON.stringify(parentElementOrClassName)})`);
 	}
 
+	// resolveImageSrc can be called frequently for the same image. To avoid unnecessary IPC,
+	// use an InMemoryCache.
+	const resolvedImageSrcCache = new InMemoryCache();
```

Comment:
On mobile, this change prevents flickering when images are downloaded from the sync target. In particular, updating one image would (often) cause CodeMirror to reload the `ImageWidget`s for all visible images. The reloaded image would remain blank until the image SRC could be fetched over IPC through `resolveImageSrc`.